crates/starkgate-manager-client/src/interfaces/manager.rs

```diff
@@ -0,0 +1,98 @@
+use async_trait::async_trait;
+use ethers::{
+    contract::ContractError,
+    prelude::abigen,
+    providers::Middleware,
+    types::{TransactionReceipt, H160, U256},
+};
+
+use utils::errors::Error;
+
+type Address = H160;
```

Comment:
Tbh I'd remove these aliases, they do not contribute much to the readability (on the contrary you have to check what's the actual type).

Reply:
But isn't Address more readable than H160?